index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into databases that index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
```

Comment:
> conforming document here only applies to the registered media types we have and normative statements constraining their members, right?

Yes.

> I feel like this might actually be describing vc+ld+json, since a +json processor would not care about these members being present....

Well, by that definition a +json processor doesn't care about any member being present, 'cause it's JSON and anything that is syntactically valid goes. I would expect that people don't think "any JSON goes" since we do talk about and link to "conforming document".

What we are trying to say here is that "Application-specific JSON processing". If you have an idea of how to say that better, please suggest some language.